### DISCUSSION TITLE
Showing all layers for crib sheet

### DISCUSSION THREAD

**Discussion by alexnharvey** [nickcoutsos/keymap-editor#199] [Ideas]
Is there a way (or could there be) to show some or all of the keyboard layers on one page which could then be printed and used as a reminder crib sheet?

**Comment by nickcoutsos** [comment]
Hi

It doesn't exist right now, sadly, and I don't have a rough timeline of when it might. There's another discussion where I was sharing/soliciting ideas for a future implementation https://github.com/nickcoutsos/keymap-editor/discussions/141 and I had started some experimentation with something that would at least allow for manually exporting an image from the browser but life events have gotten in the way.

If you haven't looked at it already the [keymap-drawer](https://github.com/caksoylar/keymap-drawer/) project is able to process ZMK keymaps into diagrams and can be hooked into GitHub actions to benefit from automation.

**Reply by alexnharvey** [reply]
Thanks Nick. https://github.com/caksoylar/keymap-drawer/ doesn't display properly for the keyboard (https://github.com/climent/atreyu), although your editor handles it ok.
I get `Error while drawing SVG from keymap YAML`. I'll try to fix that when I get a chance. 

Thanks for all your hard work, it has made the transition to ZMK so much easier for me.
